### DISCUSSION TITLE
Use GlobalAttention as GraphLevelClassifier

### DISCUSSION THREAD

**Discussion by vlatorre847** [pyg-team/pytorch_geometric#3644] [Q&A]
I wanted to use the GlobalAttention layer as graph level classifier instead of the more standard global_mean_pool or global_max_pool. However I noticed that its implementation is more complex than the aforementioned methods.

Is there any example I can use as reference? I searched the examples but I could not find anything.

**Comment by rusty1s** [comment]
We have an example of `GlobalAttention` [here](https://github.com/pyg-team/pytorch_geometric/blob/master/benchmark/kernel/global_attention.py) :)
